CR Transition admin drafts (#47)
w3c/did-wg: admin/CR-March-2021/CRACAnnouncement.txt

To: xueyuan, Cc: coralie, comm team, denis, jennyliang@w3.org, DID WG Chairs, Manu

[DRAFT] Decentralized Identifiers (DIDs) v1.0 is a W3C Candidate Recommendations (Call for Implementations)
-----------------------------------------------------------------------------------------------------------

Dear Advisory Committee Representative,
Chairs,

> I am pleased to announce that the following documents is now a Candidate Recommendations:

Reviewer: ```suggestion
I am pleased to announce that the following document is now a Candidate Recommendation:
```

To: xueyuan, Cc: coralie, comm team, denis, jennyliang@w3.org, DID WG Chairs, Manu

> [DRAFT] Decentralized Identifiers (DIDs) v1.0 is a W3C Candidate Recommendations (Call for Implementations)

Reviewer 2: Not sure if this plural is a typo or deliberate:
```suggestion
[DRAFT] Decentralized Identifiers (DIDs) v1.0 is a W3C Candidate Recommendation (Call for Implementations)
```

-----------------------------------------------------------------------------------------------------------

> Dear Advisory Committee Representative,

Reviewer 2: ```suggestion
Dear Advisory Committee Representatives,
```
